static/docs/commands-reference/config.md

```diff
@@ -181,6 +181,10 @@ $ dvc config core.loglevel debug
 
 Add an S3 remote and set it as the project default:
 
+Before,adding a new remote be sure to login into AWS services and follow
```

Comment:
use `>` before each line to make it a "note"

Reply:
ok